external/builder/test-fixtures_esprima.mjs

```diff
@@ -48,6 +48,10 @@ files.forEach(function (expectationFilename) {
   if (out !== expectation) {
     errors++;
 
+    if (process.env.OVERWRITE) {
+      fs.writeFileSync(expectationFilename, out);
```

Comment:
I committed this by accident, but it's quite handy so probably I can leave it. It makes it possible to simply run `OVERWRITE=true node ./external/builder/test-fixtures_esprima.mjs` to regenerate the expected output in case of a change.

Reply:
I'm fine with doing this, with the following two things addressed:
 - Add a code comment explaining what it's for, and how to use this, to help future readers of this code actually use the new functionality.
 - Add the same functionality in https://github.com/mozilla/pdf.js/blob/master/external/builder/test-fixtures.mjs as well, for consistency across these tests.